gbpservice/nfp/configurator/lib/schema.py

```diff
@@ -81,24 +81,10 @@
               'stitching_cidr': "",
               'stitching_interface_index': "",
               'provider_mac': "",
-              'stitching_mac': "",
```

Comment:
These changes are fine. Wonder if any of the unit test cases needs to be modified. Can we check that.